automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/user/FreeTrialSpec.scala

```diff
@@ -51,21 +56,33 @@ class FreeTrialSpec extends FreeSpec with BeforeAndAfterEach with Matchers with
     }
 
     "Enabled" - {
```

Comment:
The latter part of the `Enabled` test case already covers the case where an `Enrolled` user sees the expected page, correct?

Reply:
I think so ... should this be labeled `Enrolled` instead?

Reply:
Splitting it into `Enabled` and `Enrolled` could make it easier to see what's being tested from a quick glance but the potential set-up complexity is probably not worth it.